Backend/Models/Messages/CommandMessage.cs

```diff
@@ -127,5 +134,35 @@ private void ProcessMute()
             SendAccept();
             CLogger.Event("User Muted: " + _target);
         }
+        private void ProcessRemember()
+        {
+            int param1, param2;
+            if (!int.TryParse(_from, out param1) || !int.TryParse(_to, out param2))
+            {
+                SendRefuse("Invalid limits provided");
+                return;
+            }
+
+            int fromId = Math.Min(param1, param2);
```

Comment:
I don't get this logic, why do we need to do this?

Reply:
I was confused on why this function wasn't working, turned out when i designed it zman m3 alprotocol i made the `from` be the latest message and the `to` be the earlier messages (so the `from` number was larger than the `to` number)...
But when i was testing it now, i was accidently inverting the parameters (`from` is smaller, `to` is bigger), so i was torn on which order to use and i just thought why not both

Reply:
Oh no lala choose one and stick to it this way is too confusing. Just make sure it's documented

Reply:
done